packages/paste-core/components/chat-log/src/ChatAttachmentLink.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Anchor} from '@twilio-paste/anchor';
+import type {AnchorProps} from '@twilio-paste/anchor';
+import type {BoxElementProps} from '@twilio-paste/box';
+import {Truncate} from '@twilio-paste/truncate';
+
+export interface ChatAttachmentLinkProps extends AnchorProps {
+  children: string;
+  element?: BoxElementProps['element'];
+}
+
+const ChatAttachmentLink = React.forwardRef<HTMLElement, ChatAttachmentLinkProps>(
+  ({children, href, element = 'CHAT_ATTACHMENT_LINK', ...props}, ref) => {
+    return (
+      <Anchor href={href} ref={ref} element={element} {...props}>
+        <Truncate title={children}>{children}</Truncate>
```

Comment:
thought: I can't test this right now because the chromatic build is failing on a conflict with main, but I'm curious if this works properly. Not sure if we wrap Anchor with Truncate or Truncate with Anchor.

Leaving this as a note to myself to review this when the builds pass, or feel free to chime in.

Reply:
In the truncate docs we tell people to use truncate inside of anchor: https://paste.twilio.design/components/truncate/#truncate-url

Reply:
In addition to what @shleewhite said, it's also this truncate-inside-anchor composition in the stories for `Truncate` itself. Lemme know if it's still something you want to chat about tho @TheSisb!